stable/admin/templates/_helpers.tpl

```diff
@@ -369,16 +369,44 @@ networking.gke.io/load-balancer-type: "Internal"
 Renders the labels for all resources deployed by this Helm chart
 */}}
 {{- define "admin.resourceLabels" -}}
-app: {{ template "admin.fullname" . }}
+{{- include "admin.labels" (dict "Root" . "ExtraLabels" .Values.admin.resourceLabels ) -}}
+{{- end -}}
+
+{{- define "admin.labels" -}}
+{{- if not .Root}}{{fail "<nuodb.env> Root is required"}}{{end}}
```

Comment:
Where does this get defined? I don't see this in the Helm documented as a [built-in object](https://helm.sh/docs/chart_template_guide/builtin_objects/), and I don't see it being defined here unless I missed something.

Under what circumstances would this check fail?